reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -334,12 +336,16 @@ def generate_xy_plane(n_pix):
         -------
         xy_plane : arr
             Array describing xy plane in space.
-            Shape (n_pix, n_pix, 3)
+            Shape (3, n_pix**2)
         """
-        # See how meshgrid and generate coordinates functions used
-        # https://github.com/geoffwoollard/compSPI/blob/stash_simulate/src/simulate.py#L96
+        axis_pts = np.arange(-n_pix // 2, n_pix // 2)
+        grid = np.meshgrid(axis_pts, axis_pts)
+
+        xy_plane = np.zeros((3, n_pix**2))
+
+        for d in range(2):
+            xy_plane[d, :] = grid[d].flatten()
```

Comment:
We can change the initial shape to be that which is returned. Then we don't need the transpose at the end.